app/src/main/java/com/example/cats_or_dogs2/TableViewActivity.java

```diff
@@ -15,8 +23,13 @@ protected void onCreate(Bundle savedInstanceState) {
 
         configureBackButton();
 
+        animal_table = findViewById(R.id.recycler_table);
```

Comment:
Since the `TableViewActivity` is responsible for displaying the `RecyclerView`, all the `RecyclerView` initialization code should go in here as this is the screen that will manage it.